README.md

```diff
@@ -1,3 +1,25 @@
 # RAISE Data
 
-This exact scope of this repo is still evolving, but currently it is being used as a home for backend code used to process / wrangle RAISE data.
+This repo includes backend code used to process / wrangle RAISE data. We utilize an event-driven produce-consumer architecture backed by SNS and SQS. The consumers are realized via simple processors that pull messages from SQS and transform data in a consumer specific manner (e.g. for enclaves, dashboards, etc.). The repo also contains loaders that can be used by devs to "replay" historical messages (typically this is used to backfill data for new consumers).
```

Comment:
Typo: "produce-consumer" --> "producer-consumer".

Reply:
Good catch -- fixed in [this comment](https://github.com/openstax/raise-data/pull/8/commits/b67ae922bb75d7840f9f270f92dfda98e1d79d1c).